### PR TITLE
Add instruction doc for maintainers

### PR DESCRIPTION
**Pull request by toastwaffle**
Saves me delving through my shell history to get the right sequence of commands!

Couple of small README updates, mostly for consistent style.

### CODE REVIEW

File: maintaining.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+$ sudo pacman -S --needed arch-install-scripts e2fsprogs
+
+$ sudo ./build-arch-gce
+
+$ gcloud auth login
+
+$ gcloud config set project arch-linux-gce
+
+$ gsutil cp "arch-v${DATE}.tar.gz" gs://arch-linux-gce
```
Comment (lcastelli) [inline maintaining.md:21]: I generally use gs://arch-linux-gce-work for this.

File: maintaining.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+$ gcloud compute images create "arch-v${DATE}" \
+    --source-uri="gs://arch-linux-gce/arch-v${DATE}.tar.gz" \
+    --guest-os-features=VIRTIO_SCSI_MULTIQUEUE \
+    --family=arch
```
Comment (lcastelli) [inline maintaining.md:26]: I also add (the somewhat redundant) "--description="Arch-Linux built on ${DATE}."

**Review by lcastelli — CHANGES_REQUESTED** [review]
Thanks!

File: maintaining.md (hunk 1 above)
Comment (toastwaffle) [reply maintaining.md]: Done

File: maintaining.md (hunk 2 above)
Comment (toastwaffle) [reply maintaining.md:27]: Done

**Review by lcastelli — APPROVED** [review]

**Pull request merged**